main/reference/vstorage-ref.md

```diff
@@ -0,0 +1,434 @@
+        },
+        issuer: Object @Alleged: stATOM issuer#board00689 {},
+        issuerName: 'stATOM',
+        proposedName: 'stATOM',
+      },
+      'ibc/6831292903487E58BF9A195FDDC8A2E626B3DF39B88F4E7F41C935CADBAF54AC': {
+        brand: Object @Alleged: USDC_grv brand#board04542 {},
+        denom: 'ibc/6831292903487E58BF9A195FDDC8A2E626B3DF39B88F4E7F41C935CADBAF54AC',
+        displayInfo: {
+          assetKind: 'nat',
+          decimalPlaces: 6,
+        },
+        issuer: Object @Alleged: USDC_grv issuer#board00443 {},
+        issuerName: 'USDC_grv',
+        proposedName: 'USC Coin',
+      },
+      'ibc/BA313C4A19DFBF943586C0387E6B11286F9E416B4DD27574E6909CABE0E342FA': {
+        brand: Object @Alleged: ATOM brand#board05557 {},
+        denom: 'ibc/BA313C4A19DFBF943586C0387E6B11286F9E416B4DD27574E6909CABE0E342FA',
+        displayInfo: {
+          assetKind: 'nat',
+          decimalPlaces: 6,
+        },
+        issuer: Object @Alleged: ATOM issuer#board02656 {},
+        issuerName: 'ATOM',
+        proposedName: 'ATOM',
+      },
+      'ibc/F2331645B9683116188EF36FC04A809C28BD36B54555E8705A37146D0182F045': {
+        brand: Object @Alleged: USDT_axl brand#board01744 {},
+        denom: 'ibc/F2331645B9683116188EF36FC04A809C28BD36B54555E8705A37146D0182F045',
+        displayInfo: {
+          assetKind: 'nat',
+          decimalPlaces: 6,
+        },
+        issuer: Object @Alleged: USDT_axl issuer#board06445 {},
+        issuerName: 'USDT_axl',
+        proposedName: 'Tether USD',
+      },
+      ubld: {
+        brand: Object @Alleged: BLD brand#board0566 {},
+        denom: 'ubld',
+        displayInfo: {
+          assetKind: 'nat',
+          decimalPlaces: 6,
+        },
+        issuer: Object @Alleged: BLD issuer#board0592 {},
+        issuerName: 'BLD',
+        proposedName: 'Agoric staking token',
+      },
+      uist: {
+        brand: Object @Alleged: IST brand#board0257 {},
+        denom: 'uist',
+        displayInfo: {
+          assetKind: 'nat',
+          decimalPlaces: 6,
+        },
+        issuer: Object @Alleged: IST issuer#board0223 {},
+        issuerName: 'IST',
+        proposedName: 'Agoric stable token',
+      },
+    }
+```
+
+## boardAux
+
+The keys under `published.boardAux` are board IDs.
+Here we show a handful.
+
+```
+    [
+      'board00282',
+      'board0074',
+      'board01744',
+    ]
+```
+
+The data are auxiliary info about objects in the board;
+for example, displayInfo of brands, including assetKind.
+
+```js
+    {
+      board0074: {
+        allegedName: 'Zoe Invitation',
+        displayInfo: {
+          assetKind: 'set',
+        },
+      },
+      board01744: {
+        allegedName: 'USDT_axl',
+        displayInfo: {
+          assetKind: 'nat',
+          decimalPlaces: 6,
+        },
+      },
+    }
+```
+
+## vstorage: provisionPool
+
+`published.provisionPool.governance` gives current values of governed params.
+See also Inter Protocol governance.
+
+    {
```

Comment:
triple-quotes here and for `.metrics`, please. I think it looks much better that way. Smaller, too!